content/blog/01_mapmaking.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Map Making"
+date = 2021-05-01
+
+extra.image = "assets/images/blog/mapmaking.png"
++++
+
+Map Making 101
+
+When a builder decides that they want to make a map, they first of all, head to our (whitelisted) build server found at <code>build.nucleoid.xyz</code>
+
+They choose a game and create a map using a handy command.
+
+<br>
+<img class="header_image" src="https://iili.io/Kg34kl.jpg">
+
+One of our builders is creating a [KotH (King of the Hill)](https://github.com/NucleoidMC/koth) map using [Plasmid](https://github.com/NucleoidMC/plasmid). Plasmid is our custom game library which allows us to create minigames, and maps for those minigames, with ease.
+
+Plasmid allows our builders to access a <code>/map</code> command. The map command allows the swift creation of maps, editing, and exporting for use in production. 
```

Comment:
Code blocks should also be possible through markdown

Reply:
The markdown codeblocks aren't rendererd. I think its a zola bug.

Reply:
It was missing in the CSS- it should be fine now to use markdown.